base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -1488,8 +1623,14 @@ where
                 .master_key_manager
                 .get_next_spend_and_script_key()
                 .await?;
-            let sender_offset_private_key = PrivateKey::random(&mut OsRng);
+            // TODO: Hansie - supply the correct recovery_byte here
```

Comment:
Perhaps just clarify why this isn't the correct one?

Reply:
That was a note to self and I have already implemented the code, just need to remove the comment. Thanks